### ISSUE TITLE
rest position for mechs

### ISSUE THREAD

**Issue by najlitarvan** [tsunamayo/Starship-EVO#6091]
ability to define a "rest" position for mech, this position would could activate in either of two cases: player is not controlling the mech or the mech computer would get a signal input
if i am to give an example subnautica's prawn suit will sit down when above water and not being piloted by player
sitting prawn:
![image](https://github.com/user-attachments/assets/b6195d79-afc4-4ef5-89c3-28d5e6f95342)
standing prawn:
![image](https://github.com/user-attachments/assets/5933d298-bdd6-4134-9721-18bf1ead93ca)
 
other example could be the b1 droids from star wars
stored form:
![image](https://github.com/user-attachments/assets/dd0283eb-8648-44e7-bf9f-4406e5ac8bd6)
normal form:
![image](https://github.com/user-attachments/assets/22bf0e72-e312-43af-9959-028e963f9ef1)

could be neat to design compact holding cells, i can imagine it being a neat addition to mech dropships

**Comment by tsunamayo** [comment]
@najlitarvan Yes I want to do it but the problem is that I am still not sure the current anim logic will stay, as people hate it and might needs a total wipeout.
